packages/camera/camera_android_camerax/android/build.gradle

```diff
@@ -57,16 +57,17 @@ android {
         }
     }
 
-    lintOptions {
+    lint {
```

Comment:
[`lintOptions`](https://developer.android.com/reference/tools/gradle-api/7.3/com/android/build/api/dsl/LintOptions) is deprecated!